### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by Lucky390163** [moneytoo/Player#411] [Ideas]
Can u add display clock/battery in now plaing.

**Comment by moneytoo** [comment]
What exactly do you mean by that? On phones, when controls are displayed, there's also visible status bar which normally shows clock and battery.

**Reply by Lucky390163** [reply]
Yes. Exactly

**Comment by Lucky390163** [comment]
And please also add an option to take screenshots and use their own subtitles with a download feature from the open subtitle site.

**Comment by Lucky390163** [comment]
I mean use own style of subtitle customization of the app.

**Reply by moneytoo** [reply]
https://github.com/moneytoo/Player#how-do-i-change-subtitle-font-size-or-color

**Comment by Lucky390163** [comment]
I mean inbuilt subtitle feature.

**Reply by moneytoo** [reply]
Why?

**Comment by Lucky390163** [comment]
Because device subtitles are dirty in look Because of background black colour.

**Reply by moneytoo** [reply]
This is already configurable in captioning preferences.

**Comment by Lucky390163** [comment]
Can u add a option of secure folder lock with fingertips and face unlock.

**Comment by Lucky390163** [comment]
I strongly need this.
